### PR TITLE
feat: Create card banner

### PR DESCRIPTION
**Pull request by poom-kitti**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -12,11 +12,7 @@ export default defineComponent({
...
 <style>
 #app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
+  font-family: Inter;
+  color: #090c2f;
```
Comment (new5558) [inline src/App.vue:16]: `#090c2f` color code is redundant. Can we have a separate ts file to import color from?

File: src/commons/UI/BaseButton.vue [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<template>
+  <button class="button bg-primary rounded-lg text-white font-body">
+    <slot></slot>
+  </button>
+</template>
+
+<script>
+export default {};
```
Comment (new5558) [inline src/commons/UI/BaseButton.vue:8]: ??

File: src/commons/UI/BaseButton.vue [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  </button>
+</template>
+
+<script>
+export default {};
+</script>
+
+<style scoped>
+.button {
+  box-shadow: 0px 2px 0px rgba(0, 0, 0, 0.043);
```
Comment (new5558) [inline src/commons/UI/BaseButton.vue:13]: why not use tailwind shadow?

File: src/commons/UI/BaseIcon.vue [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+<script>
+export default {
+  props: {
+    iconName: {
+      type: String,
+      default: "box"
+    },
+    width: {
+      type: [Number, String],
```
Comment (new5558) [inline src/commons/UI/BaseIcon.vue:25]: why prop can be both number and string. Should we only use number?

File: src/modules/home/components/CardBanner.vue [hunk 5]
```diff
@@ -1,24 +1,89 @@
...
+    CalendarIcon,
+    ClockIcon
+  },
   setup() {
-    const img = "https://picsum.photos/200";
+    const eventTitle = "Event Title";
+    const tags = ["Tech", "Math", "Science"];
+    const eventDescription =
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer velnisi eu ipsum elementum sollicitudin. Suspendisse aliquam arcu quis cursus cursus. Nulla sollicitudin ut felis sit amet blandit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer vel nisi euipsum elementum sollicitudin. Suspendisse aliquam arcu quis cursus          cursus. Nulla sollicitudin ut felis sit amet blandit.";
+    const img = "https://picsum.photos/1000";
```
Comment (new5558) [inline src/modules/home/components/CardBanner.vue:73]: please change name to Image url

File: src/modules/home/components/CardBanner.vue [hunk 6]
```diff
@@ -1,24 +1,89 @@
...
+    const eventDate = "Sun, 14 Feb - 16 April";
+    const eventTime = "10:00 - 18:00";
```
Comment (new5558) [inline src/modules/home/components/CardBanner.vue:75]: This is strange. what if we have an event that has a different time and different date

File: src/modules/home/components/CardBanner.vue [hunk 7]
```diff
@@ -27,4 +92,11 @@ export default {
 .event-container {
   box-shadow: 0px 2px 20px rgba(0, 0, 0, 0.08);
```
Comment (new5558) [inline src/modules/home/components/CardBanner.vue:93]: box shadow from tailwind

**Review by new5558 — DISMISSED** [review]

**Comment by new5558** [comment]
Please add preview on home so that I can review the layout

File: src/modules/home/components/CardBanner.vue (hunk 6 above)
Comment (new5558) [reply src/modules/home/components/CardBanner.vue:75]: @Ukkabanana

File: src/App.vue (hunk 1 above)
Comment (new5558) [reply src/App.vue:16]: https://github.com/tailwindlabs/discuss/issues/119

Comment (new5558) [reply src/App.vue:16]: https://www.telerik.com/blogs/passing-variables-to-css-on-a-vue-component

Comment (new5558) [reply src/App.vue:16]: NVM: I find the better way to do this
https://tailwindcss.com/docs/adding-base-styles
@poom-kitti

**Review by Seqlet — APPROVED** [review]
LGTM

**Pull request merged**